Lib/fontTools/varLib/mutator.py

```diff
@@ -308,6 +312,16 @@ def instantiateVariableFont(varfont, location, inplace=False):
 				addidef = any(op.startswith("GETVARIATION") for op in instructions)
 				if addidef:
 					break
+		# set macOS rendering bit (bit6) if set_macos_rendering_bit parameter is set to True
```

Comment:
This comment line can be removed

Reply:
https://github.com/fonttools/fonttools/pull/1518/commits/01c2c082cb9f0cf395509025a1245c269910d1a5